app/models/video.rb

```diff
@@ -1,2 +1,4 @@
 class Video < ApplicationRecord
+  #optional: trueはデータが取れるようになったら外す
```

Comment:
```suggestion
  # TODO: optional: trueはデータが取れるようになったら外す
```

アノテーションコメント
https://qiita.com/taka-kawa/items/673716d77795c937d422